docs/standard/microservices-architecture/secure-net-microservices-web-applications/authorization-net-microservices-web-applications.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: About authorization in .NET microservices and web applications
-description: .NET Microservices Architecture for Containerized .NET Applications | About authorization in .NET microservices and web applications
+description: Securing .NET Microservices and Web Applications | Get an overview of the main authorization options in ASP.NET Core applications - role-based and policy-based.
```

Comment:
Securing => Secure

Reply:
Thinking on SEO, changed all titles on this chapter to this pattern "Security in .NET Microservices and Web Applications - {page description}"